docs/mmteb/points.md

```diff
@@ -28,6 +28,7 @@
 | cassanof          |              | 6           |          |                     | 1          | 1              |           |                |          |               |
 | taidnguyen        |              | 14          |          |                     |            |                |           |                |          |               |
 | orionw         |              |            |          |                     |      2      |              |           |                |          |               |
+| rbroc             |       2      |      2      |          |                     |      2      |              |           |                |          |               |
```

Comment:
I would add +4 here for adding dataset (first non-translated Italian classification). 

The points for running models are intended for running models on the whole benchmark so those will have to be cut. New task is intended for adding new types of task formulations (e.g. classification vs. retrieval). 

Will you also add a point for the review (me/Kennethenevoldsen)